src/backend/src/app.js

```diff
@@ -53,6 +52,19 @@ export async function init({ version, rootKey }) {
     coreStorage: () => new RAM(),
   })
 
+  await manager.start()
+
+  rnBridge.app.on('pause', async (pauseLock) => {
+    log('App went into background')
+    await manager.stop()
+    pauseLock.release()
+  })
```

Comment:
does `messagePort` need to be stopped here too?

Reply:
no